corehq/apps/userreports/transforms/specs.py

```diff
@@ -127,3 +128,57 @@ def transform_function(values):
             return delimiter.join(translated_values_list)
 
         return transform_function
+
+
+class HyperLinkTransform(Transform):
+    type = TypeProperty("hyperlink")
+    link_text = StringProperty(required=True)
+
+    def get_transform_function(self):
+        link_text = mark_safe(self.link_text)
```

Comment:
what errors is `mark_safe` helping avoid? would it be possible to add tests that document them?